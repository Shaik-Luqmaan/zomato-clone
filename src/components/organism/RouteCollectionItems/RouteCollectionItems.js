import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import { Grid } from "@material-ui/core";
import RouteCollection from "../../molecules/RouteCollections/RouteCollection"
import { Link } from "react-router-dom";

const useStyles = makeStyles({
    root: {
      padding: "20px 40px 80px 70px",
    },
});

function RouteCollectionItems(props) {
    const classes = useStyles();
    // const {CollectionList} = props;

    return (
        <Grid container alignItems="stretch" justify="space-evenly" className={classes.root} spacing={1}>
        
        <Grid item md={3} sm={6} xs={12}>
          <Link to="/hyderabad/delivery" style={{ textDecoration: 'none' }}>
              <RouteCollection img={"https://b.zmtcdn.com/webFrontend/64dffaa58ffa55a377cdf42b6a690e721585809275.png"}
                title={"Order Food Online"}/>
          </Link>
        </Grid>

        <Grid item md={3} sm={6} xs={12}>
          <Link to="/hyderabad/dine-out" style={{ textDecoration: 'none' }}>
             <RouteCollection img={"https://b.zmtcdn.com/webFrontend/95f005332f5b9e71b9406828b63335331585809309.png"}
                title={"Go out for a meal"} />
          </Link>
        </Grid>

        <Grid item md={3} sm={6} xs={12}>
          <Link to="/hyderabad/night-life" style={{ textDecoration: 'none' }}>
              <RouteCollection img={"https://b.zmtcdn.com/webFrontend/8ff4212b71b948ed5b6d2ce0d2bc99981594031410.png"}
                title={"Night Life & CLubs"} />
          </Link>
        </Grid>

        <Grid item md={3} sm={6} xs={12}>
           <RouteCollection img={"https://b.zmtcdn.com/webFrontend/b256d0dd8a29f9e0623ecaaea910534d1585809352.png"}
              title={"Zomato Pro"}/>
          
        </Grid>

        </Grid>
          
      );
}

// const CollectionList = [
//   {
//       title: "Order Food Online",
//       img: "https://b.zmtcdn.com/webFrontend/64dffaa58ffa55a377cdf42b6a690e721585809275.png"
//   },
//   {
//       title: "Go out for a meal",
//       img: "https://b.zmtcdn.com/webFrontend/95f005332f5b9e71b9406828b63335331585809309.png"
//   },
//   {
//       title: "Night Life",
//       img: "https://b.zmtcdn.com/webFrontend/8ff4212b71b948ed5b6d2ce0d2bc99981594031410.png"
//   },
//   {
//       title: "Zomato Pro",
//       img: "https://b.zmtcdn.com/webFrontend/b256d0dd8a29f9e0623ecaaea910534d1585809352.png"
//   }

// ]

export default RouteCollectionItems
