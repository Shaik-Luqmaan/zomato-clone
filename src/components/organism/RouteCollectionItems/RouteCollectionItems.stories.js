import React from "react";
import RouteCollectionItems from "./RouteCollectionItems"

export default {
    title: "organisms/RouteCollectionItems",
    component: RouteCollectionItems,
}


const Template = ({...args})=>(
    <RouteCollectionItems {...args} />
);


export const Primary = Template.bind({});
Primary.args = {
   
}