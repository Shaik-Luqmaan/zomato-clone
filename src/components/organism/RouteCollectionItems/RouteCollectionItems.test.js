import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
import "@testing-library/jest-dom";
import RouteCollectionItems from "./RouteCollectionItems";
import renderer from "react-test-renderer";

it("renders snapshot correctly", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <RouteCollectionItems />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders RouteCollectionItems without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(
    <BrowserRouter>
      <RouteCollectionItems />
    </BrowserRouter>,
    div
  );
});