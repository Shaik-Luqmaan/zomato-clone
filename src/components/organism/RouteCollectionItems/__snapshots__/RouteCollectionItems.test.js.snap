// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot correctly 1`] = `
<div
  className="MuiGrid-root makeStyles-root-1 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-justify-xs-space-evenly"
>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
  >
    <a
      href="/hyderabad/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "none",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <img
            alt="Route Card"
            className="makeStyles-img-3"
            src="https://b.zmtcdn.com/webFrontend/64dffaa58ffa55a377cdf42b6a690e721585809275.png"
          />
        </div>
        <div
          className="MuiGrid-root makeStyles-text-4 MuiGrid-item"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="sample"
          >
            Order Food Online
          </h6>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
  >
    <a
      href="/hyderabad/dine-out"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "none",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <img
            alt="Route Card"
            className="makeStyles-img-3"
            src="https://b.zmtcdn.com/webFrontend/95f005332f5b9e71b9406828b63335331585809309.png"
          />
        </div>
        <div
          className="MuiGrid-root makeStyles-text-4 MuiGrid-item"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="sample"
          >
            Go out for a meal
          </h6>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
  >
    <a
      href="/hyderabad/night-life"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "none",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <img
            alt="Route Card"
            className="makeStyles-img-3"
            src="https://b.zmtcdn.com/webFrontend/8ff4212b71b948ed5b6d2ce0d2bc99981594031410.png"
          />
        </div>
        <div
          className="MuiGrid-root makeStyles-text-4 MuiGrid-item"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="sample"
          >
            Night Life & CLubs
          </h6>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
  >
    <div
      className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center"
    >
      <div
        className="MuiGrid-root MuiGrid-item"
      >
        <img
          alt="Route Card"
          className="makeStyles-img-3"
          src="https://b.zmtcdn.com/webFrontend/b256d0dd8a29f9e0623ecaaea910534d1585809352.png"
        />
      </div>
      <div
        className="MuiGrid-root makeStyles-text-4 MuiGrid-item"
      >
        <h6
          className="MuiTypography-root MuiTypography-h6"
          data-testid="sample"
        >
          Zomato Pro
        </h6>
      </div>
    </div>
  </div>
</div>
`;
